From d509e486fa2a931c177292ce0cb81ceecd24c81d Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Fri, 13 Oct 2023 08:06:52 +0000
Subject: Use correct install locations

Use the correct install locations for QtAsyncio
and designer plugin.

Pick-to: 6.6
Fixes: PYSIDE-2494
Change-Id: I760e8f52f19cd1f1cdf742a15df071c4f4deffa5
Reviewed-by: Qt CI Bot <qt_ci_bot@qt-project.org>
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
---
 sources/pyside6/PySide6/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

(limited to 'sources/pyside6/PySide6/CMakeLists.txt')

diff --git a/sources/pyside6/PySide6/CMakeLists.txt b/sources/pyside6/PySide6/CMakeLists.txt
index e24fdb682..37b7a6c9c 100644
--- a/sources/pyside6/PySide6/CMakeLists.txt
+++ b/sources/pyside6/PySide6/CMakeLists.txt
@@ -111,4 +111,4 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/pyside6_global.h
         DESTINATION include/${BINDING_NAME}${pyside6_SUFFIX})
 
 install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/QtAsyncio"
-        DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
+        DESTINATION "${PYTHON_SITE_PACKAGES}/${BINDING_NAME}${pyside6_SUFFIX}")
-- 
cgit v1.2.3

